.gitignore

```diff
@@ -116,3 +117,11 @@ dmypy.json
 
 # setuptools-scm
 version.py
+
+# Ape stuff
+tests/hardhat.config.js
```

Comment:
why is this being ignored?

Reply:
The config file is written as part of the test run. It shouldn't be managed by git.

Reply:
ah, hardhat just makes this file appear?

Reply:
No, the config file is written in the `HardhatProvider.__post_init__`. If you run hardhat without a config file set it will prompt you to create one first, so I had it write a minimal config file if one doesn't already exist.